### ISSUE TITLE
Schedulers

### ISSUE THREAD

**Issue by emailweixu** [HorizonRobotics/alf#589]
During training, some hyper-parameters (e.g learning rate, entropy target, etc.) need to change with iterations. It's tedious to separately implementing the scheduling for each different hyper-parameter. We can use the method in this post to implement schedulers which can be used in the same way as an int or float: https://stackoverflow.com/questions/1117149/does-python-have-conversion-operators
